app/helpers/application_helper.rb

```diff
@@ -1,2 +1,16 @@
 module ApplicationHelper
+  def area_tag(area, classes = [])
```

Comment:
Any chance we can put this in a different helper? `ApplicationHelper` is usually reserved for methods that are used sitewide